Dockerfile.rpm

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2021 CloudTruth, Inc.
+# All Rights Reserved
+#
+# Build the image using the project directory as context.
+# Map the project directory to /cli when running the container.
+#
+
+FROM centos:8
```

Comment:
CentOS 7 might be a better target given it has a longer support window. Usually older binaries will work fine on newer releases, as well. The inverse might not be true.

Reply:
If we're going to just use the musl binary instead of building then perhaps this would not matter as much?  Not sure.